src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt

```diff
@@ -235,6 +323,15 @@ class DefaultEthereumMethods(
             chainUnsupportedMethods(chain) +
             getDrpcVendorMethods(chain) +
             getChainSpecificMethods(chain)
+
+        // add trace method for arbitrum
+        traceMethods.plus(
```

Comment:
Let's move it to the `getChainSpecificMethods` method where all specific methods are